apps/crn-server/test/data-providers/research-outputs.data-provider.test.ts

```diff
@@ -844,12 +845,13 @@ describe('ResearchOutputs data provider', () => {
         const researchOutputRequest = getResearchOutputCreateDataObject();
         const teamId = researchOutputRequest.teamIds[0];
         const researchOutputId = 'created-output-id';
+        const now = Date.now().toString();
```

Comment:
i dont think we want any extra functionality inside our data providers - they should just accept data as is and it should be the controllers responsibility to default the `addedDate` to current time